src/nativeComponents.ts

```diff
@@ -88,13 +88,21 @@ function readMetadata(elfPath: string) {
     );
   }
 
+  let platformApiVersion;
+  try {
+    platformApiVersion = findSection('app_apiversion').toString();
+  } catch (ex) {
+    platformApiVersion = 'N/A';
+  }
+
   return {
     platform,
     path: elfPath,
     data: elfData,
     appID: formatUUID(appIDData),
     buildID: `0x${Buffer.from(buildIDData).swap64().toString('hex')}`,
     family: findSection('appfamily').toString(),
+    platformCApiVersion: platformApiVersion,
```

Comment:
nit: you can simply write `platformCApiVersion,` here if you name the variable the same - also helps keep it consistent

Reply:
Done.

Reply:
You can simply write `platformCApiVersion,` rather than `platformCApiVersion: platformCApiVersion,` if you place it after `platform`.